page01_index.php

```diff
@@ -1,177 +1,171 @@
-<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
-
-
-
-</head>
-<body>
-
-<!-- Navbar (sit on top) -->
-<div class="w3-top">
-  <div class="w3-bar w3-white w3-wide w3-padding w3-card">
-    <img src="images/placeholder.jpg" width="40" height="40">
-
-    <a href="indexV1.php" class="w3-bar-item w3-button"><b>logo</b> </a>
-
-    <!-- Float links to the right. Hide them on small screens -->
-    <div class="w3-right w3-hide-small">	
-      <a href="indexV1.php" class="w3-bar-item w3-button">Home</a>
-      <a href="playlistV1.php" class="w3-bar-item w3-button">Playlist </a>	
-      <a href="contact.html" class="w3-bar-item w3-button" >Contact</a>			
-      <a href="03_add_user(settings).php" class="w3-bar-item w3-button">Settings</a>  
-      <a href="signout.php" class="w3-bar-item w3-button w3-text-red">Log out</a>
-    </div>
-  </div>
-</div>
+	<head>
+		<title>Sound Wave</title>
+		<meta charset = "utf-8">
+		<link rel = "stylesheet" type = "text/css" href = "css/fonts.css">
+		<link rel = "stylesheet" type = "text/css" href = "css/grids.css">
+		<link rel = "stylesheet" href = "css/style.css"/>
+		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
 
 
-<!-- opening image of main design  -->
-<div class="w3-display-container">
-  <img src="images/placeholder.jpg"  style="width:100%;height:800px">
+		<meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
+		<meta name = "Author" content = "Liv Amer"/>
+		<meta name = "Description" content = "Graeme's Music Database"/>
+		<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
+
 
-  <div class="w3-display-middle" >
-    <h1 class="w3-jumbo w3-animate-top">SOUNDWAVE</h1>
-    <hr class="w3-border-grey" style="margin:auto;width:40%">
-    <p class="w3-large w3-center">Feel the music</p>
-  </div>
-</div>
+	</head>
+	<body>
 
+		<!-- Navbar (sit on top) -->
+		<div class="w3-top">
+			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
+				<img src="images/placeholder.jpg" width="40" height="40">
+				<a href="index.php" class="w3-bar-item w3-button"><b>logo</b> </a>
 
-<!-- container to create more padding and margin -->
-<div class="w3-panel w3-padding-large w3-margin-left w3-margin-right w3-blue">
+				<!-- Float links to the right. Hide them on small screens -->
+				<div class="w3-right w3-hide-small">
+					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<?php
```

Comment:
added an admin feature for Graeme, so he can control his database and users on it